src/oc/nrnmpi.h

```diff
@@ -13,6 +13,7 @@ extern int nrnmpi_numprocs;       /* size of subworld */
 extern int nrnmpi_myid;           /* rank in subworld */
 extern int nrnmpi_numprocs_bbs;   /* number of subworlds */
 extern int nrnmpi_myid_bbs;       /* rank in nrn_bbs_comm of rank 0 of a subworld */
+extern int nrnmpi_subworld_change_cnt;
```

Comment:
Some documentation here would be nice since at the same place we have comments for all the other related variables

Reply:
Yes, I have added a comment there.